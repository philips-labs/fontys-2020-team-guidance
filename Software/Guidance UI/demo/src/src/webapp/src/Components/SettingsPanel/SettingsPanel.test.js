import React from 'react';
import { render, cleanup, fireEvent} from '@testing-library/react';
import ShallowRenderer from 'react-test-renderer/shallow';
import SettingsPanel from "./SettingsPanel";

afterEach(cleanup);

it('renders correctly react-test-renderer', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<SettingsPanel />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
});