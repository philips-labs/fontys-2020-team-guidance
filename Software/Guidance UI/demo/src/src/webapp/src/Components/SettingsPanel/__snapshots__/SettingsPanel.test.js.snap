// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly react-test-renderer 1`] = `
<div
  className="mainLoading"
>
  <p
    className="loading"
  >
    loading...
  </p>
</div>
`;
